examples/wifi-echo/server/esp32/main/EchoDeviceCallbacks.cpp

```diff
@@ -122,6 +116,79 @@ void EchoDeviceCallbacks::PostAttributeChangeCallback(uint8_t endpoint, EmberAfC
     {
         ESP_LOGE(TAG, "Unexpected endpoint id: %d", endpoint);
     }
+}
+
+TimerHandle_t identifyTimer;
+void IdentifyTimerCallback(TimerHandle_t handle)
+{
+    statusLED1.Animate();
+
+    // Get remaining timer count.
+    uint32_t timerCount = (uint32_t) pvTimerGetTimerID(handle);
+
+    // Decrement the timer count.
+    timerCount--;
+    vTimerSetTimerID(handle, (void *) timerCount);
+
+    // If the timer count reaches 0, just stop the timer.
+    if (timerCount == 0)
+    {
+        xTimerStop(handle, 0);
+    }
+}
+
+void IdentifyPostAttributeChangeCallback(uint8_t endpoint, uint16_t attributeId, uint8_t * value)
+{
+    ESP_LOGI(TAG, "endpoint: '0x%02x', attribute ID: '0x%04x', value: %d", endpoint, attributeId, *value);
+
+    if (attributeId != ZCL_IDENTIFY_TIME_ATTRIBUTE_ID)
+    {
+        ESP_LOGI(TAG, "Unknown attribute ID: %d", attributeId);
+        return;
+    }
+
+    // Only endpoint 1 supports the Identify cluster
+    if (endpoint == 1)
+    {
+        statusLED1.Blink(500);
+
+        // timerCount represents the number of callback execution before the we stopped the timer.
+        // value is expressed in seconds and the timer is fired every 250ms, so just multiply value by 4.
+        // Also, we want timerCount to be odd number, so the ligth state ends in the same state it starts.
```

Comment:
"light"

Reply:
I don't understand the comment about it being an "odd" number.  Do you mean an "even" number?

Reply:
Obviously I mean an even number. My English skills has failed here.